model-engine/model_engine_server/inference/post_inference_hooks.py

```diff
@@ -122,6 +124,7 @@ def handle(
             return
         response["task_id"] = task_id
         data_record = {
+            "EMITTED_AT": datetime.now(pytz.timezone("US/Pacific")).strftime("%Y-%m-%dT%H:%M:%S"),
```

Comment:
nit: should be a UTC timestamp so there's no timezone or daylight saving time information baked into the dates here

Reply:
i'm fine with using UTC but seems like we're using PST elsewhere in the codebase? 
https://github.com/scaleapi/llm-engine/blob/31c7c5a392a1265c52b7f70e8e08d20ca293f697/model-engine/model_engine_server/api/llms_v1.py#L125